anet.yml

```diff
@@ -392,6 +392,10 @@ dictionary:
         options:
           layers: GEBCO_LATEST
           format: "image/png"
+    avatar:
+      maxFileSize: 71680
+      height: 200
+      width: 200    
```

Comment:
Don't we assume that height is equal to width? We could have a single value `radius`

Reply:
Are you sure we want to assume that? If so I will change it.

Reply:
I removed this, and now it is set in the front end display component (though it is not configurable)